src/lib/agent/PolykeyAgent.ts

```diff
@@ -3,15 +3,20 @@ import fs from 'fs';
 import net from 'net';
 import path from 'path';
 import process from 'process';
-import { fork, ForkOptions } from 'child_process';
-import Polykey, { KeyManager } from '../Polykey';
 import Configstore from 'configstore';
+import { Duplex } from 'readable-stream';
 import PolykeyClient from './PolykeyClient';
 import { agent } from '../../../proto/js/Agent';
-import { Duplex } from 'readable-stream';
+import PeerInfo from '../../lib/peers/PeerInfo';
+import Polykey, { KeyManager } from '../Polykey';
+import { fork, ForkOptions } from 'child_process';
 const {
+  AddPeerRequestMessage,
```

Comment:
Why can't we just leave all these protobuf generated objects in the `agent.*`?

Also I would not call it that. It might better to call it `agentInterface` or `agentProto.*`.

That's going to be much clearer.

Reply:
yeah that might save a whole bunch of code at the top, I will change it to `agentInterface`, sounds appropriate

Reply:
done. I've also changed the others to be consistent with this naming convention.